includes/polling/os/linksys-ss.inc.php

```diff
@@ -0,0 +1,9 @@
+<?php
+// Linksys OS details
+
+$hardware = snmp_get($device, 'LINKSYS-Physicaldescription-MIB::rlPhdUnitGenParamModelName.1', '-OQv');
```

Comment:
You can switch all of these to a multi get to save single requests being made.

Also, you can move the MIB name in to the params passed to the function.

Reply:
done